src/dns_sd.cpp

```diff
@@ -86,25 +86,27 @@ init(Handle<Object> target) {
     addConstants(target);
 }
 
-inline
+NAN_INLINE
```

Comment:
Hehe... Please use keyword `inline`. As a collaborator on NAN I am strictly opposed to any application of `NAN_INLINE`. It shouldn't exist.

/cc @kkoopa